aten/src/ATen/native/quantized/affine_quantizer.cpp

```diff
@@ -80,6 +82,13 @@ void checkZeroPoints(const std::string& fn_name, Tensor zero_points) {
   }
 }
 
+void checkFloatZeroPoints(const std::string& fn_name, Tensor zero_points) {
```

Comment:
Is this needed? float values will always fit in float type.

Reply:
Yeah, can remove this check